snapcraft/internal/build_providers/_lxd/_lxd.py

```diff
@@ -365,6 +382,49 @@ def pull_file(self, name: str, destination: str, delete: bool = False) -> None:
     def shell(self) -> None:
         self._run(command=["/bin/bash"])
 
+    def _wait_for_network(self) -> None:
+        while True:
+            if self._container is None:
+                raise RuntimeError("Attempted to use container before starting.")
+            if any(
+                a["address"]
+                for a in self._container.state().network["eth0"]["addresses"]
+                if a["family"] == "inet"
+            ):
+                break
+            sleep(1)
```

Comment:
Maybe make 0.1?  I don't think that'd be unreasonable - especially if it saves me a few seconds each day :stuck_out_tongue_winking_eye:

Reply:
sure